test/opsem/muchang.gstr.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: %seabmc --horn-bv2-word-size=1 "%s" 2>&1 | %oc %s
```

Comment:
we should have multiple tests. With different word sizes. You can have multiple `RUN` commands in a test.